timelock-server/src/main/java/com/palantir/atlasdb/timelock/LockRefreshTokens.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock;
+
+import java.math.BigInteger;
+import java.util.UUID;
+
+import com.palantir.lock.LockRefreshToken;
+import com.palantir.lock.v2.LockTokenV2;
+
+public class LockRefreshTokens {
```

Comment:
should we push responsibility for conversion to the client side? the only reason we can't return `LockTokenV2` here is that it's a pain to deal with two types of refresh tokens in the transaction manager... and we can't convert the old `LockRefreshToken` to a `LockTokenV2` because BigInteger might have > 128 bits and thus won't fit in a UUID.